include/ttmlir/Dialect/TTNN/Transforms/Passes.td

```diff
@@ -28,4 +28,12 @@ def TTNNLayout : Pass<"ttnn-layout", "::mlir::ModuleOp"> {
   }];
 }
 
+def TTNNWorkarounds : Pass<"ttnn-workaround", "::mlir::ModuleOp"> {
+  let summary = "Apply workarounds to the IR.";
+  let description = [{
+    This pass applies necessary TTNN workarounds to the IR in order to create
+    a valid TTNN flatbuffer.
```

Comment:
I think you can omit `in order to create a valid TTNN flatbuffer.` since we can target emitC path, and not emit any fb.